march_rqt_input_device/src/march_rqt_input_device/input_device_controller.py

```diff
@@ -0,0 +1,54 @@
+import rospy
+import std_msgs.msg
+
+from march_shared_resources.msg import Error, GaitInstruction
+
+
+class InputDeviceController(object):
+    def __init__(self):
+        self.instruction_gait_pub = rospy.Publisher('/march/input_device/instruction', GaitInstruction, queue_size=10)
+
+        self.error_pub = rospy.Publisher('/march/error', Error, queue_size=10)
+        self._ping = rospy.get_param('~ping_safety_node', True)
+
+        if self._ping:
+            self._alive_pub = rospy.Publisher('/march/input_device/alive', std_msgs.msg.Time, queue_size=10)
+            self._alive_timer = rospy.Timer(rospy.Duration(0.05), self._timer_callback)
+
+    def shutdown_plugin(self):
+        if self._ping:
+            self._alive_timer.shutdown()
+            self._alive_pub.unregister()
+
+    def _timer_callback(self, event):
+        self._alive_pub.publish(event.current_real)
+
+    def publish_rocker_switch(self, string):
+        rospy.logdebug('Mock Input Device published rocker switch {dr}'.format(dr=string))
+        self.instruction_gait_pub.publish(GaitInstruction(std_msgs.msg.Header(stamp=rospy.Time.now()),
+                                                          GaitInstruction.ROCKER_SWITCH, '', string))
```

Comment:
I would not name it the `ROCKER_SWITCH` command, since this is very dependent on how the hardware is currently implemented, we could in the future choose it with some other kind of input. So I would name it `INCREASE_STEP_SIZE` and `DECREASE_STEP_SIZE` or something like that. Which more resembles the method instead of the hardware.